### ISSUE TITLE
[Domework] - Feature

### ISSUE THREAD

**Issue by hisRoyalty** [PGamerX-Legacy/issues#4]
These are some tiny winey bit of adjustments or “suggestions” that could be implemented in DomeWork.

- Make prefix case sensitive for the mobile users gang.
- Make it so that if the message includes numerals or something, make it return and say to use calculator (for d!homework text).
- Dashboard (maybe), or atleast a website

**Comment by unknown123456789149** [comment]
1.Not sure cause we will probaly soon shift all message commands to slash commands
2.Good idea in my opinion, but it could change when shifting to slash commands
3.We actually already have a website, pgamerx.com, but maybe we will make a dashboard later but finish mine first 😈

**Comment by hisRoyalty** [comment]
1. Didn’t think about that.
2. Makes sense.
3. Ofcource


Also more suggestions-
I know this is a homework bot, but maybe add some quizzes and stuff? Or maybe, a brief explanation on some topics, like for example, algebra, or maybe something more advanced about history, and yes this might take a lot of time.

Secondly, English is an important subject, so maybe a command linking to a dictionary? I’m sure they’re tons of packages on those which you could use.

**Comment by pgamerx** [comment]
> 1. Didn’t think about that.
> 2. Makes sense.
> 3. Ofcource
> 
> 
> Also more suggestions-
> I know this is a homework bot, but maybe add some quizzes and stuff? Or maybe, a brief explanation on some topics, like for example, algebra, or maybe something more advanced about history, and yes this might take a lot of time.
> 
> Secondly, English is an important subject, so maybe a command linking to a dictionary? I’m sure they’re tons of packages on those which you could use.

Domework wasn't made as a mainstream bot, that's why it lies below other bots and will not get new features very quick.

These suggestions will be added in our task list but don't get your hopes high for getting them within days/few weeks. It will take time

**Comment by hisRoyalty** [comment]
Yes, and upgrading it to fully v13 is not the easiest task.

**Comment by pgamerx** [comment]
> Yes, and upgrading it to fully v13 is not the easiest task.

Exactly, slash commands (interactions) do not support images as of now and that's why we will have to wait.

**Comment by unknown123456789149** [comment]
> Yes, and upgrading it to fully v13 is not the easiest task.

Copy that

**Comment by unknown123456789149** [comment]
> > Yes, and upgrading it to fully v13 is not the easiest task.
> 
> Exactly, slash commands (interactions) do not support images as of now and that's why we will have to wait.

Didnt even think about that yet, will they even support that?

**Comment by Peiprjs** [comment]
Well I just had it use an image link in my test
But it's kinda harder

**Comment by pgamerx** [comment]
> Well I just had it use an image link in my test
> But it's kinda harder

URL is bad

**Comment by Peiprjs** [comment]
URL bad
